homeassistant/components/eufylife_ble/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "domain": "eufylife_ble",
+  "name": "EufyLife",
```

Comment:
Maybe we should indicate that its Bluetooth so they get the idea that they need working bluetooth

Reply:
Out of curiosity, why do you think this is needed? I don't see Bluetooth mentioned in the name of other Bluetooth integrations like Inkbird and Oral-B.

Reply:
This is oralb:
<img width="472" alt="Screenshot 2023-01-16 at 1 27 52 PM" src="https://user-images.githubusercontent.com/663432/212779773-d073856c-e69e-4964-9a0f-b086465351a0.png">

This is eufy:
<img width="471" alt="Screenshot 2023-01-16 at 1 28 01 PM" src="https://user-images.githubusercontent.com/663432/212779818-a335a38a-682c-491d-a9df-dff160852326.png">

How does the user know which one to pick?
<img width="480" alt="Screenshot 2023-01-16 at 1 27 57 PM" src="https://user-images.githubusercontent.com/663432/212779831-8e12115b-4ad7-4e8c-85cb-d52ce75c9b88.png">

Reply:
After this PR is merged, I'm planning to submit a follow-up PR to rename the existing "eufy" integration to "EufyHome". That integration is only for devices sold under the EufyHome product line, and my new integration is for the EufyLife product line. So, the use will choose based on the product line of the device they are trying to integrate.